### DISCUSSION TITLE
Config Question

### DISCUSSION THREAD

**Discussion by kunaldas926** [aws-solutions/workload-discovery-on-aws#410] [General]
I want to understand how the solution is using config service.

**Comment by svozza** [comment]
Workload Discovery deploys a [Config Aggregator](https://docs.aws.amazon.com/config/latest/developerguide/aggregate-data.html) to collect information about the resources in your accounts. When you [import a region](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/import-a-region.html) in the Workload Discovery UI we provide you with CloudFormation templates to deploy in that account and region: the regional template enables Config in the region and authorizes that region to allow it to send data to the WD Config Aggregator.

The discovery process queries the Config aggregator every fifteen minutes to calculate what has changed in the imported accounts and regions since it last ran. This data is persisted in the Neptune database and the data is exposed with a GraphQL API. The WD UI queries this API when you are creating diagrams.

**Reply by JoBruns** [reply]
Workload Discovery is not collecting the AWS resource inventory data on its own. Rather, it delegates the collection to its service **AWS Config** that is doing exactly this. 

Before you can explore your service landscape, you must ensure (among some other things mentioned [here](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/prerequisites.html)) that AWS Config is running in each region you want to have inspected. The data from all these region is periodically being aggregated and staged for subsequent analysis. You can adjust this frequency depending on your use case.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
